lizard_auth_server/tests/test_views_api_v2.py

```diff
@@ -340,15 +338,14 @@ def test_duplicate_username(self):
         form = mock.Mock()
         form.cleaned_data = self.user_data
         response = self.view.form_valid(form)
-        self.assertEqual(500, response.status_code)
+        self.assertEqual(409, response.status_code)
 
     def test_duplicate_username_http_response(self):
         client = Client()
         params = {'username': 'pietje',
                   'email': 'nietpietje@example.com',
                   'first_name': 'pietje',
-                  'last_name': 'pietje',
-              }
+                  'last_name': 'pietje', }
```

Comment:
Deze stijl ben ik niet zo voor (danwel, deze stijl zie je bij onze code verder niet). Of geen komma en acollade op dezelfde regel of wel een komma en dan acollade op de volgende regel.

Reply:
Ok, aangepast!